packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker/create-transform-job.ts

```diff
@@ -173,6 +181,23 @@ export class SageMakerCreateTransformJob extends sfn.TaskStateBase {
     };
   }
 
+  private renderModelClientOptions(config: ModelClientOptions): { [key: string]: any } {
+    const retries = config.invocationsMaxRetries;
+    if (retries? (retries < 0 || retries > 3): false) {
+      throw new RangeError(`invocationsMaxRetries should be between 0 and 3, not ${retries}.`);
+    }
+    const timeout = config.invocationsTimeout?.toSeconds();
+    if (timeout? (timeout < 1 || timeout > 3600): false) {
+      throw new RangeError(`invocationsTimeout should be between 1 and 3600 seconds, not ${timeout}.`);
```

Comment:
```suggestion
      throw new Error(`invocationsTimeout should be between 1 and 3600 seconds. Received: ${timeout}.`);
```

really awesome!! - I think we should throw `Error` as we shifted away from throwing `RangeError` a while back.